dash_bio/Ideogram.py

```diff
@@ -87,15 +92,15 @@ class Ideogram(Component):
 
 General case to specify specific chromosomes:
 Ex: chromosomes=['1', '2']
-- dataDir (string; optional): Absolute or relative URL of the directory containing data needed to draw banded chromosomes.
+- dataDir (string; default 'https://unpkg.com/ideogram@1.5.0/dist/data/bands/native/'): Absolute or relative URL of the directory containing data needed to draw banded chromosomes.
 You will need to set up your own database to grab data from a custom database.
-- organism (string | number; optional): Organism(s) to show chromosomes for. Supply organism's name as a string (e.g., "human") or
+- organism (string | number; default 'human'): Organism(s) to show chromosomes for. Supply organism's name as a string (e.g., "human") or
 organism's NCBI Taxonomy ID (taxid, e.g., 9606) to display chromosomes from a single
 organism, or an array of organisms' names or taxids to display chromosomes from multiple
 species.
 - localOrganism (dict; optional): Provide local JSON organism into this prop from a local user JSON file.
 DataDir must not be initialized.
-- homology (optional): Used to compare two chromosomes with each other.
+- homology (dict; optional): Used to compare two chromosomes with each other.
```

Comment:
```suggestion
- homology (dict; optional): Used to compare two chromosomes from different organisms.
```
Repeating the definition of homology... Alternatively, we may end the description after "chromosomes."